### DISCUSSION TITLE
Error using appwindowfactory in MATLAB

### DISCUSSION THREAD

**Discussion by Liuqi2002519** [alexisInSAR/EZ-InSAR#109] [Help]
Hi everyone! I meet a problem when I tried to open EZ_InSAR on matlabR2019b,it told me "Functionality not supported with figures created with the uifigure function. For more information, see Graphics Support in App Designer.".I tried to install EZ_InSAR again, but mistake appeared again. Can someone tell me why this mistake appear? Thanks!

**Comment by alexisInSAR** [comment]
Hi, 

EZ-InSAR requires MATLAB >2020b. 

Kind regards, 

Alexis

**Reply by Liuqi2002519** [reply]
Thank you for your answer. I have successfully run EZ-InSAR on matlab2022b. If it is not presumptuous, I would like to ask you to mention the matlab version issue during the installation process. This only requires a small reminder. Because I did not see this prompt during the installation process.Thanks！
